### ISSUE TITLE
New next.js builds and app router

### ISSUE THREAD

**Issue by seagullua** [kuma-ui/kuma-ui#320]
The latest next.js builds have removed `appDir` from the `experimental` section in `next.config.js`. 

Now it shows a warning on launch:
```
- warn Invalid next.config.js options detected: 
- warn: The value at .experimental has an unexpected property, appDir, which is not in the list of allowed properties
```

However, if I remove it, kuma stops adding CSS to app folder components.

**Comment by kfranqueiro** [comment]
If I'm reading right, this is only in [13.4.20-canary-13](https://github.com/vercel/next.js/releases/tag/v13.4.20-canary.13) and later, i.e. not in any full releases quite yet?

It looks like they've seriously replaced `experimental.appDir` references with hard-coded `true` values, including their invocation of `findPagesDir`, which is what Kuma UI also calls. (I suppose it remains to be seen whether they end up altering this function's signature later.)

Seems like Kuma UI would need a new way to detect app-dir-by-default in Next 13.4.20+, vs. pages-dir-by-default in earlier releases?

Relevant next PRs: https://github.com/vercel/next.js/pull/54785, https://github.com/vercel/next.js/pull/54788

**Comment by seagullua** [comment]
@kfranqueiro I think the best approach will be to support both folders simultaneously, as lots of teams are migrating from pages to apps, and they may have pages in both folders simultaneously.

**Comment by taishinaritomi** [comment]
will work on this.

**Comment by poteboy** [comment]
Should be resolved in #337
